package.json

```diff
@@ -163,6 +163,11 @@
                 "type": "boolean",
                 "description": "Enable logging of the Debug Adapter Protocol.",
                 "default": true
+              },
+              "runDebug": {
+                "type": "boolean",
+                "description": "Enable running of the Debug Server",
+                "default": false
```

Comment:
It should default to true, by default we will have the extension kick off the server.  In development environments will be where we want to disable that at times.

Reply:
Or change the name of it so that default to false makes sense.  Either way as long as the default behavior is to download and start the backend DAPodil server.

Reply:
changed `runDebug` to `disableDapServer`. This was the best way to make sure the dap server was ran if var was not assigned in the launch.json